virttest/libvirt_xml/devices/tpm.py

```diff
@@ -70,5 +77,68 @@ def __init__(self, virsh_instance=base.base.virsh):
                                    parent_xpath='/',
                                    tag_name='encryption',
                                    attribute='secret')
+            accessors.XMLElementNest('active_pcr_banks',
+                                     libvirtxml=self,
+                                     parent_xpath='/',
+                                     tag_name='active_pcr_banks',
+                                     subclass=self.ActivePCRBanks,
+                                     subclass_dargs={
+                                         'virsh_instance': virsh_instance})
             super(self.__class__, self).__init__(virsh_instance=virsh_instance)
             self.xml = '<backend/>'
+
+        class ActivePCRBanks(base.base.LibvirtXMLBase):
+            """
+            Tpm active_pcr_banks xml class.
+
```

Comment:
Perhaps you can add what one simple activePCRBanks looks like here?

Reply:
Do you mean xml example like following?
<active_pcr_banks>
<sha256 />
</active_pcr_banks>

Reply:
@yanqzhan  yes